.github/OWNERS

```diff
@@ -8,6 +8,7 @@ reviewers:
   - nikhita
   - parispittman
 approvers:
+  - sig-contribex-approvers
   - castrojo
```

Comment:
IMO if we are adding an alias, we can remove explicit approvers from here.

cc @castrojo @mrbobbytables @cblecker

Reply:
Done